backend/app/controllers/spree/admin/refunds_controller.rb

```diff
@@ -8,6 +8,22 @@ class RefundsController < ResourceController
 
       rescue_from Spree::Core::GatewayError, with: :spree_core_gateway_error
 
+      def create
```

Comment:
Hm, I think we actually can't use `create.after` anymore because we need to use `create_and_perform` instead of `save` + `perform!`.